### PR TITLE
Upgrade attachment champs length

### PR DESCRIPTION
**Pull request by LaBisquerie**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      |  The number of characters available for naming files when they are added in the back office is too short for displaying them on the front, for example, to name a technical manual.
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | https://github.com/PrestaShop/PrestaShop/pull/37598
| Sponsor company   | creabilis.com
| How to test?      | Go to Catalog > Files and add a new file with a filename greater than 32 characters, and a filename greater than 128 characters.

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hello @LaBisquerie!

This is your first pull request on [autoupgrade](https://github.com/PrestaShop/autoupgrade/) repository of the PrestaShop project.

Thank you, and welcome to this Open Source community!

**Review by Hlavtox — APPROVED** [review]
Good job @LaBisquerie :-)

**Review by SharakPL — APPROVED** [review]

**Review by M0rgan01 — APPROVED** [review]

**Review by ga-devfront — APPROVED** [review]

**Review by AureRita — APPROVED** [review]
Hi @LaBisquerie 

Thank you for your PR, I tested it and it seems to works as you can see : 

image

image


Tested from : 
1.7.8.10 to 8.2 
8.0.4 to 8.2 
8.2 to 9.0.0
8.0.4 to 9.0.0

Because the PR seems to works as expected, It's QA :heavy_check_mark: 

Thank you

**Comment by M0rgan01** [comment]
wait until the PR is merged https://github.com/PrestaShop/PrestaShop/pull/37598

**Pull request merged**
